### ISSUE TITLE
full download size for cached repo packages displayed in prompt

### ISSUE THREAD

**Issue by rmarquis** [rmarquis/pacaur#424]
The _Download Size_ of binary packages is always displayed, even though the packages are actually present in cache.

(cached and not cached)
```
Repo Packages (27):

Name                            Old Version  New Version  Download Size

...
community/python2-netaddr                    0.7.18-3          1.05 MiB

Repo Download Size:    8.45 MiB
```

The size should be hidden (or set to 0.00), and the total Repo Download Size should be computed accordingly.

For comparison, here is pacman output:

(not cached)
```
$ sudo pacman -S community/python2-netaddr
resolving dependencies...
looking for conflicting packages...

Package (1)                New Version  Net Change  Download Size

community/python2-netaddr  0.7.18-3       5.52 MiB       1.05 MiB

Total Download Size:   1.05 MiB
Total Installed Size:  5.52 MiB
```

(cached)
```
$ sudo pacman -S community/python2-netaddr
resolving dependencies...
looking for conflicting packages...

Package (1)                New Version  Net Change

community/python2-netaddr  0.7.18-3       5.52 MiB

Total Installed Size:  5.52 MiB
```

**Comment by rmarquis** [comment]
This is actually an issue happening when `PKGEXT` is changed from the default `.pkg.tar.xz`.
When using `'.pkg.tar` in makepkg.conf (to disable package compression for speed), the function GetBuiltPkg() can't find the packages in the pacman cache anymore, and the prompt supposes they aren't cached at all.

**Comment by rmarquis** [comment]
A bit hacky, but fixed in 56500a28f6216f21c0dc4affdcc08b1d06b41ec8.
